Source/WebCore/platform/graphics/skia/SkiaRenderingSettings.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2024 Igalia S.L.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ * You should have received a copy of the GNU Library General Public License
+ * along with this library; see the file COPYING.LIB.  If not, write to
+ * the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301, USA.
+ */
+
+#include "config.h"
+#include "SkiaRenderingSettings.h"
+
+#if USE(COORDINATED_GRAPHICS) && USE(SKIA)
+#include <wtf/NumberOfCores.h>
+#include <wtf/text/StringToIntegerConversion.h>
+
+namespace WebCore {
+
+bool SkiaRenderingSettings::forceUseCpuRendering()
+{
+    static bool* forceUseCpuRendering = nullptr;
+    if (!forceUseCpuRendering) {
+        const char* enableCPURendering = getenv("WEBKIT_SKIA_ENABLE_CPU_RENDERING");
+        forceUseCpuRendering = new bool(enableCPURendering && strcmp(enableCPURendering, "0"));
+    }
+
+    return *forceUseCpuRendering;
+}
+
+unsigned SkiaRenderingSettings::numberOfCpuPaintingThreads()
+{
+    static unsigned* numberOfCpuPaintingThreads = nullptr;
```

Comment:
Use std::optional<unsigned> instead of a pointer.

Reply:
Done, nicer, thanks.